### PR TITLE
Aquifer Maps and Shapefiles

### PR DESCRIPTION
**Pull request by countable**

### CODE REVIEW

File: app/frontend/src/common/assets/js/leaflet-areaselect.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
+L.AreaSelect = L.Class.extend({
```
Comment (stephenhillier) [inline app/frontend/src/common/assets/js/leaflet-areaselect.js]: need a blurb about the attribution of this file and the License (I think this is MIT licensed if I'm looking at the right repo)

File: app/frontend/src/aquifers/components/View.vue [hunk 2]
```diff
@@ -13,7 +13,7 @@
 */
 
 <template>
-  <b-card class="container p-1">
+  <b-card class="container container-wide p-0 card-container pb-3" :class="{ 'p-4': editMode }">
```
Comment (stephenhillier) [inline app/frontend/src/aquifers/components/View.vue]: can you double check this works as intended (second `class` attribute)?

Second thing is that I think the container-wide class has been removed in our code base.  `container` now has an extra media query for 1080p screens with a max width of 1400px (App.vue).

**Comment by stephenhillier** [comment]
Just need to add some attribution info for the file that was added, otherwise looks great

File: app/frontend/src/aquifers/components/View.vue (hunk 2 above)
Comment (byshine) [reply app/frontend/src/aquifers/components/View.vue]: yes it does work as intended.

File: app/frontend/src/common/assets/js/leaflet-areaselect.js (hunk 1 above)
Comment (byshine) [reply app/frontend/src/common/assets/js/leaflet-areaselect.js]: where do you want the blurb? also what should it say?

Comment (stephenhillier) [reply app/frontend/src/common/assets/js/leaflet-areaselect.js]:
```suggestion
/** 
 * leaflet-areaselect plugin by Jonatan Heyman
 * https://github.com/heyman/leaflet-areaselect
 * Source code made available under MIT license.
 */

L.AreaSelect = L.Class.extend({
```

**Comment by stephenhillier** [comment]
Closing this PR since the other aquifers PR includes these commits.

**Pull request closed without merging**